client/views/network.tpl

```diff
@@ -6,6 +6,12 @@
 	data-nick="{{nick}}"
 	data-options="{{tojson serverOptions}}"
 >
+	<span class="collapse-network-tooltip tooltipped tooltipped-e tooltipped-no-touch" aria-label="Collapse network…" data-alt-label="Cancel">
+		<button class="collapse-network" aria-label="Collapse network…" data-id="{{id}}"
```

Comment:
Remove `…` from `Collapse network…`. Ellipsis in menus often means that there are further options or required input from the user. I'm sure there are better reads than this one, but for example: https://ux.stackexchange.com/a/9545/90032

We've been doing better at this lately, that wasn't the case before. For example, we correctly have "Join a channel…", "Change nick…" vs. "Send message", etc.

Reply:
Also, did you mean "Expand" instead of "Cancel"? I'd actually vote for "Collapse" instead of "Collapse network" but no big deal. Or even no tooltip actually, we don't have one for other collapse/expand buttons (previews, status messages, and in a further extent, "More details" on error loading page).

(Kinda odd that in "alt mode", tooltip `aria-label` will be "Expand" and button `aria-label` will be "Collapse". I know it's not your PR, just an observation, but we ought to fix this someday 😅)

Reply:
Yeah that's... Copy paste error. Good catch, will fix it. Would need to keep the aria but happy to remove the tooltip if you don't want it.

Reply:
Yeah I'd say remove it, but if there are strong opinions against that, then I don't mind.

Reply:
Alright, I removed it. Also, updating the aria-label when we expand/collapse now.